src/routes/cities.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const { City, validate } = require('../models/city');
```

Comment:
There is no "validate" function exported from ../models/city (rather validateCity). wrong name of the function here, clearly, leads to broken post/put requests

Reply:
My bad. Changed it after testing all cities functionality.
Fixed.